view/frontend/templates/oneclick/magiccart.phtml

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
You should never use ObjectManager directly.

Instead, inject ViewModel to the view.